workers/unity/Assets/Fps/Scripts/GameLogic/Respawning/SpawnPoints/SpawnPointIndicator.cs

```diff
@@ -4,11 +4,12 @@ namespace Fps
 {
     public class SpawnPointIndicator : MonoBehaviour
     {
-        private readonly Vector3 boxDimensions = new Vector3(1, 0, 1);
+        private static readonly Color transparentLightGreen = new Color(.1f, 1f, .1f, .8f);
+        public static readonly Vector3 BoxDimensions = new Vector3(.5f, 2f, .5f);
```

Comment:
Why is this public now?

Reply:
I was confused by this also, as nothing references it...but I just realised it's because I have an external tool I made to allow exporting spawn points to Unreal, and it gets the box dimensions to determine how much to offset the Unreal spawn point height by. It's pretty unnecessary, and I can hardcode the height offset. Will fix.